web/docs/project/env-vars.md

```diff
@@ -2,61 +2,165 @@
+import { env } from 'wasp/server'
+
+console.log(env.SOME_VAR_NAME)
 ```
 </TabItem>
 <TabItem value="ts" label="TypeScript">
 
 ```ts
-console.log(process.env.SOME_VAR_NAME)
+import { env } from 'wasp/server'
+
+console.log(env.SOME_VAR_NAME)
 ```
 </TabItem>
 </Tabs>
 
-Check below on how to define them.
+Read more about the `env` object in the [API reference](#server-env-vars-1).
+
+### Wasp Server Env Vars
+
+#### General Configuration
+
+<EnvVarsTable>
+    <EnvVar name="DATABASE_URL" type="String" isRequired={true} note="The URL of the PostgreSQL database you want your app to use." />
+    <EnvVar name="WASP_WEB_CLIENT_URL" type="URL" isRequired={true} note="Used as your client URL. The server needs to know about it to properly configure Same-Origin Policy (CORS) headers." />
+    <EnvVar name="WASP_SERVER_URL" type="URL" isRequired={true} note="Used as your server URL. The server needs it to properly redirect users when logging in with OAuth providers like Google or GitHub." />
+    <EnvVar name="JWT_SECRET" type="String" isRequired={true} note={
+      <span>Needed to generate secure tokens. <a href="https://jwtsecret.com/generate" target="_blank" rel="noreferrer">Generate</a> a random string at least 32 characters long.
+      </span>
+    } />
+    <EnvVar name="PORT" type="Integer" isRequired={false} defaultValue="3001" note="This is where the server listens for requests." />
+</EnvVarsTable>
+
+#### SMTP Email Sender
+
+If you are using `SMTP` as your email sender, you need to provide the following environment variables:
+
+<EnvVarsTable>
+    <EnvVar name="SMTP_HOST" type="String" isRequired={true} note="The SMTP server host." />
+    <EnvVar name="SMTP_PORT" type="Integer" isRequired={true} note="The SMTP server port." />
+    <EnvVar name="SMTP_USERNAME" type="String" isRequired={true} note="The SMTP server username." />
+    <EnvVar name="SMTP_PASSWORD" type="String" isRequired={true} note="The SMTP server password." />
+</EnvVarsTable>
+
+#### SendGrid Email Sender
+
+If you are using `SendGrid` as your email sender, you need to provide the following environment variables:
+
+
+<EnvVarsTable>
+    <EnvVar name="SENDGRID_API_KEY" type="String" isRequired={true} note="The SendGrid API key." />
+</EnvVarsTable>
+
+
+#### Mailgun Email Sender
+
+If you are using `Mailgun` as your email sender, you need to provide the following environment variables:
+
+<EnvVarsTable>
+    <EnvVar name="MAILGUN_API_KEY" type="String" isRequired={true} note="The Mailgun API key." />
+    <EnvVar name="MAILGUN_DOMAIN" type="String" isRequired={true} note="The Mailgun domain." />
+    <EnvVar name="MAILGUN_API_URL" type="URL" isRequired={false} note={
+      <span>Useful if you want to use the EU API endpoint (<code>https://api.eu.mailgun.net</code>).</span>
+    } />
+</EnvVarsTable>
+
+#### Auth
+
+You can use this env while developing to skip email verification. This is useful when you are testing the email sending functionality in development:
```

Comment:
You are talking here about a single specific env var? That is a bit weird because below we have a whole table. Feels like you are talking about the whole table. Maybe put it all in the note for that one instead?

Reply:
Did this ever happen (I don't see a rocket :rocket: )?

Reply:
Yep, it happened, sorry 🙋‍♂️